### ISSUE TITLE
Specifying plugins from CLI

### ISSUE THREAD

**Issue by louiscenter** [stackcss/sheetify-postcss#4]
Is it possible to specify which `postcss` plugins you'd like `sheetify-postcss` to use from the command line?

Currently I have the following: `-t [ sheetify/transform -u sheetify-postcss ]`.
Is there any way to tell `sheetify-postcss` to use `postcss-cssnext`?

**Comment by kristoferjoseph** [comment]
Yes, how would one pass plugins to `sheetify-postcss` in a bash script for instance?
i tried:
`-t [ sheetify/transform -u sheetify-postcss -p postcss-extend ]`
`-t [ sheetify/transform -u [ sheetify-postcss -p postcss-extend ] ]`
etc. but never got it to work

Any pointers?

Thank you kindly
